source/getting-started/tutorials/timing-manager/index.md

```diff
@@ -0,0 +1,381 @@
+3. Completion of the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial
+4. Completion of the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial
+
+This tutorial expands on the code created in the [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial, and uses commands created in the [Profiling Tasks](/getting-started/tutorials/profiling-tasks/index.md) tutorial. Both must be completed before this tutorial.
+
+## Introduction
+
+In motor control applications, it is important for the timing of both sensor data acquisition and control task execution to remain consistent. In its default configuration, the AMDC may allow some jitter (timing incosistency) between sensor measurement time and control loop timing. This tutorial shows how the AMDC's [Timing Manager](/firmware/arch/timing-manager.md) peripheral can be configured to eliminate this jitter.
+
+This tutorial provides code that allows users to experiment with the configuration of the AMDC [Timing Manager](/firmware/arch/timing-manager.md). The effects of this experimentation will be observed through the lens of the previous [Voltage Source Inverter](/getting-started/tutorials/vsi/index.md) tutorial.
+
+## Scheduling and Synchronizing
+
+Through the [Timing Manager](/firmware/arch/timing-manager.md), the AMDC is capable of synchronizing sensor data collection and task execution to the PWM carrier wave. This is desirable to eliminate electromagnetic noise in motor drives (for example, switching harmonics in the phase current measurements) because the inverter does not switch at the peak and trough of the PWM carrier.
+
+Additionally, updates to the PWM duty ratio can be precisely synced with the timing manager, and are by default.
+
+### Timing Manager Modes
+
+Every `X` PWM periods (where the user specifies `X` using the function `timing_manager_set_ratio()`), the AMDC will collect data from the sensors. By default, the [Timing Manager](/firmware/arch/timing-manager.md) is set to `Legacy Mode`, in which the AMDC OS will run tasks independent of whether the sensors have completed data collection.
+
+![](images/tmLegacySimple.svg)
+
+Alternatively, when configured in `Post-Sensor Mode`, the [Timing Manager](/firmware/arch/timing-manager.md) will instruct the AMDC OS to wait to execute tasks until the sensor data collection is complete. With proper configuration, this can be used to ensure the following:
+
+![](images/tmPostSensorSimple.svg)
+
+1. the task has new sensor data and
+2. the time elapsed between when the sensor data is acquired and when the task is run is consistent and jitter-free.
+3. the pwm duty ratio is updated at a consistent rate (every X pwm cycles)
+
+### AMDC Actions Synchronized by the Timing Manager
+
+There are multiple factors that affect when and how fast control tasks run:
+- User set `TASK_NAME_UPDATES_PER_SEC`
+- PWM Frequency
+- User Event Ratio
```

Comment:
Again, help the user out. Add couple words here and link to the correct place in the timing manager article and tell us that it will be referred to as `EVENT_RATIO` in this tutorial